<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinDisplayClass</class>
 <widget class="QMainWindow" name="WinDisplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinDisplay</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLDrawer" name="glWidget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>160</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStep">
       <property name="text">
        <string>Step -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestart">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnLoadFile">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDrawer</class>
   <extends>QWidget</extends>
   <header>gldrawer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="WinDisplay.qrc"/>
 </resources>
 <connections/>
</ui>
